app/models/offering.rb

```diff
@@ -0,0 +1,13 @@
+class Offering < ApplicationRecord
+  has_one :loan
+  has_many :investments
+  has_many :invested_investments, -> { where status: 'invested' },
```

Comment:
Again feel like this could be better as a strait up association and a separate scope.  That is chained when called.